packages/ReactDagEditor/src/built-in/RectComponent.tsx

```diff
@@ -1,19 +1,7 @@
 import * as React from "react";
-import { usePropsAPI } from "../hooks/usePropsAPI";
 import { ICanvasNode } from "../models/node";
-import { IPropsAPI } from "../props-api/IPropsAPI";
 import { isNodeEditing } from "../utils/state";
 
-const getChangeHandler = (node: ICanvasNode, propsAPI: IPropsAPI) => (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
We do want the inline editing in built-in default node.

Reply:
This means the data must have a field named `comment` and it is not type safe here
Opened a issue for future solution #36